ui/src/app/store/pod/slice.ts

```diff
@@ -228,6 +228,71 @@ const podSlice = createSlice({
     ) => {
       state.errors = action.payload;
     },
+    pollLxdServer: {
```

Comment:
This action uses `get_projects` as the websocket method, but I've split it into its own action instead of making `getProjects` more flexible. The separation is a bit cleaner and the intention makes a bit more sense, imo.

Reply:
I think that makes sense.